pawn/Resources/Callbacks/OnPlayer/OnPlayerText.pwn

```diff
@@ -8,6 +8,11 @@
  * @param playerid Id of the player who typed the text.
  * @param text The text the player typed.
  */
+ new QuitJokes[] = {
```

Comment:
Why do you think that?

It needs to be above first usage, not immediately before the message. Please move it to before the comment block.